resources/conf/log_config.toml

```diff
@@ -27,4 +27,6 @@ logLevel = "INFO"
 [accessLogs]
 enable = false
 logfile = "/tmp/envoy.access.log" # This file will be created inside router container.
```

Comment:
shall we change this to /dev/stdout too. I think printing it in this location is not useful at all.